
This one has to be checked..given i think it's not updated the previous CakePHP update.

--- app/webroot/css/cake.generic.css.orig	2012-09-20 15:53:24.716684032 +0200
+++ app/webroot/css/cake.generic.css	2012-09-20 14:00:58.772522542 +0200
@@ -26,7 +26,7 @@
 	background: #003d4c;
 	color: #fff;
 	font-family:'lucida grande',verdana,helvetica,arial,sans-serif;
-	font-size:90%;
+	font-size:12px;
 	margin: 0;
 }
 a {
@@ -69,7 +69,10 @@
 	margin: 0 12px;
 }
 p {
-	margin: 0 0 1em 0;
+	margin-bottom: 8px;
+}
+em {
+	font-weight: bold;
 }
 
 /** Layout **/
@@ -143,6 +146,7 @@
 	border-bottom:2px solid #555;
 	text-align: left;
 	padding:4px;
+	white-space: nowrap;
 }
 th a {
 	display: block;
@@ -165,13 +169,34 @@
 	background: #f9f9f9;
 }
 td.actions {
-	text-align: center;
+	text-align: right;
 	white-space: nowrap;
+	width:5%;
 }
 table td.actions a {
 	margin: 0px 6px;
 	padding:2px 5px;
 }
+th.actions {
+	text-align:center;
+}
+td.short {
+	width:5%;
+	white-space: nowrap;
+}
+
+/* added */
+div .events table tr:hover, div .events table tr.altrow:hover, div .attributes table tr:hover, div .attributes table tr.altrow:hover{
+	background-color: #F5F5D7;
+}
+
+table tr.altrow {
+	background: #f5f5f5;
+}
+table tr:nth-child(2n) {
+	background: #f5f5f5;
+}
+/* /added */
 
 /* SQL log */
 .cake-sql-log {
@@ -261,9 +286,9 @@
 	width: 95%;
 }
 fieldset {
-	border: none;
+	border: 1px solid #ccc;
 	margin-bottom: 1em;
-	padding: 16px 10px;
+	padding: 16px 20px;
 }
 fieldset legend {
 	color: #e32;
@@ -271,8 +296,9 @@
 	font-weight: bold;
 }
 fieldset fieldset {
-	margin-top: 0;
-	padding: 10px 0 0;
+	margin-top: 0px;
+	margin-bottom: 20px;
+	padding: 16px 10px;
 }
 fieldset fieldset legend {
 	font-size: 120%;
@@ -306,26 +332,36 @@
 }
 label {
 	display: block;
-	font-size: 110%;
+	font-size: 120%;
 	margin-bottom:3px;
 }
+div.forminfo {
+	display: block;
+	float: right;
+	text-align: right;
+	font-weight:normal;
+	background-color: #fcfcb1;
+	padding: 2px 2px 2px 2px;
+	border: 1px dashed #e8e882;
+}
+
 input, textarea {
 	clear: both;
-	font-size: 140%;
+	font-size: 110%;
 	font-family: "frutiger linotype", "lucida grande", "verdana", sans-serif;
 	padding: 1%;
 	width:98%;
 }
 select {
 	clear: both;
-	font-size: 120%;
+	font-size: 110%;
 	vertical-align: text-bottom;
 }
 select[multiple=multiple] {
 	width: 100%;
 }
 option {
-	font-size: 120%;
+	font-size: 110%;
 	padding: 0 3px;
 }
 input[type=checkbox] {
@@ -340,9 +376,7 @@
 input[type=radio] {
 	float:left;
 	width:auto;
-	margin: 6px 0;
-	padding: 0;
-	line-height: 26px;
+	margin: 0 3px 7px 0;
 }
 .radio label {
 	margin: 0 0 6px 20px;
@@ -564,7 +598,8 @@
 pre {
 	color: #000;
 	background: #f0f0f0;
-	padding: 15px;
+	padding: 10px;
+	margin-bottom: 10px;
 	-moz-box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
 	-webkit-box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
 	box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
